HomeWork/src/main/java/io/Main.java

```diff
@@ -0,0 +1,13 @@
+package io;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+
+        FileLogger logger = new FileLogger();
+        FileLoggerConfigurationLoader loader = new FileLoggerConfigurationLoader();
+
```

Comment:
Did you test your logger? Looks like that you've created a logger and a configuration but didn't run it.